tests/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+file(GLOB test_cases CONFIGURE_DEPENDS "*.cpp")
+
+# set(test_cases 
```

Comment:
I guess we can remove the comment now :)

Reply:
Haha thanks for catching this. I have removed the comments